pom.xml

```diff
@@ -27,14 +34,63 @@
+        <mockito.version>1.10.19</mockito.version>
+        <joda-time.version>2.9.5</joda-time.version>
 
-        <tds.config.client.version>0.0.1-SNAPSHOT</tds.config.client.version>
         <tds.common.version>0.0.1</tds.common.version>
         <tds-common-legacy.version>0.0.1</tds-common-legacy.version>
-        <tds.student.client.version>0.0.1-SNAPSHOT</tds.student.client.version>
 
         <maven.compiler.plugin.version>3.5.1</maven.compiler.plugin.version>
-        <mockito.version>1.10.19</mockito.version>
-        <joda-time.version>2.9.5</joda-time.version>
+        <maven.jar.plugin.version>2.3.2</maven.jar.plugin.version>
+        <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
+        <maven.source.plugin.version>2.1.2</maven.source.plugin.version>
+        <maven.toolchains.plugin.version>1.1</maven.toolchains.plugin.version>
+        <publish.package.name>tds.config</publish.package.name>
     </properties>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>${maven.compiler.plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-release-plugin</artifactId>
+                    <version>${maven.release.plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-source-plugin</artifactId>
+                    <version>${maven.source.plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-toolchains-plugin</artifactId>
+                    <version>${maven.toolchains.plugin.version}</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <artifactId>maven-release-plugin</artifactId>
+                <configuration>
+                    <tagNameFormat>@{project.version}</tagNameFormat>
+                    <useReleaseProfile>false</useReleaseProfile>
+                    <releaseProfiles>release</releaseProfiles>
+                    <autoVersionSubmodules>true</autoVersionSubmodules>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <distributionManagement>
+        <repository>
+            <id>bintray-smarter-app-open-test-system</id>
+            <name>smarter-app-open-test-system</name>
+            <url>https://api.bintray.com/maven/smarter-app/open-test-system/${publish.package.name}/;publish=1</url>
+        </repository>
+    </distributionManagement>
```

Comment:
Created a "tds.config" package in Bintray and configured releases to publish to the package.